src/components/Dashboard/FilingHistoryList.vue

```diff
@@ -627,16 +636,24 @@ export default {
     },
 
     /**
-     * Function to return a boolean if this specific filing is future affective.
-     *
+     * Whether this specific filing is a future affective COA.
      * @param filingType The type of the filing in the history list.
      * @param status The status of the filing in the history list.
-     * @returns A boolean indicating if the filing is future effective.
      */
     isCoaFutureEffective (filingType: string, status: string): boolean {
-      return (this.isBComp() &&
+      return (
+        this.isBComp() &&
         filingType === FilingTypes.CHANGE_OF_ADDRESS &&
-        status === FilingStatus.PAID)
+        this.isPaid(status)
+      )
+    },
+
+    /**
+     * Whether this specific filing is paid.
+     * @param status The status of the filing in the history list.
+     */
+    isPaid (status: string): boolean {
+      return (status === FilingStatus.PAID)
```

Comment:
Simple and clean! 

What do we stand on specifying `@returns` in the DocBlocs?

Reply:
I removed/omitted the `@returns` line because it's covered by the main description and doesn't appear in my mouse-over help anyway. Does your mileage vary? (YMMV)